sw/device/tests/clkmgr_external_clk_src_for_sw_test.c

```diff
@@ -29,14 +29,14 @@ typedef struct expected_count_info {
 } expected_count_info_t;
 
 expected_count_info_t count_infos[kDifClkmgrMeasureClockUsb + 1] = {
-  {479, 1}, {239, 1}, {119, 1}, {499, 1}, {239, 6}};
+  {479, 1}, {239, 1}, {119, 1}, {479, 1}, {239, 6}};
```

Comment:
so the issue with external clock is that when we switch to it, everything is based on 1 frequency.
That means we can't really have both 100MHz/96MHz. So for the purposes of this test I chose 96MHz, but of course that breaks the normal measurement case, so you probably need to split this into two options.